requirements_dev.txt

```diff
@@ -52,6 +52,8 @@ typing_extensions
 boto3
 botocore
 google-cloud-storage
+google-cloud-compute
```

Comment:
Do we need google-auth here as well?

Reply:
It's a required dependency of google-cloud-compute

Reply:
Nice